### PR TITLE
moved includes docs from /guides to /deploy

### PR DESCRIPTION
**Pull request by himeshsiriwardana**
## Purpose
$subject

### CODE REVIEW

**Review by hwupathum — APPROVED** [review]

File: en/identity-server/7.0.0/docs/deploy/security/keystores/manage-ca-signed-certificates-in-a-keystore.md [hunk 1]
```diff
@@ -1 +1 @@
-{% include "../../../../../../includes/guides/encryption/manage-ca-signed-certificates-in-a-keystore.md" %}
+{% include "../../../../../../includes/deploy/security/keystores/index.md" %}
```
Comment (hwupathum) [inline en/identity-server/7.0.0/docs/deploy/security/keystores/manage-ca-signed-certificates-in-a-keystore.md:1]: Is this correct?

**Review by hwupathum — CHANGES_REQUESTED** [review]

File: en/identity-server/7.0.0/docs/deploy/security/asymmetric-encryption/index.md [hunk 2]
```diff
@@ -1 +1 @@
-{% include "../../../../../../includes/guides/encryption/asymmetric-encryption.md" %}
+{% include "../../../../../../includes/deploy/security/asymmetric-encryption.md" %}
```
Comment (hwupathum) [inline en/identity-server/7.0.0/docs/deploy/security/asymmetric-encryption/index.md:1]: includes/deploy/security/asymmetric-encryption/index.md

Comment (himeshsiriwardana) [reply en/identity-server/7.0.0/docs/deploy/security/asymmetric-encryption/index.md:1]: fixed with: https://github.com/wso2/docs-is/pull/4839

File: en/identity-server/7.0.0/docs/deploy/security/keystores/manage-ca-signed-certificates-in-a-keystore.md (hunk 1 above)
Comment (himeshsiriwardana) [reply en/identity-server/7.0.0/docs/deploy/security/keystores/manage-ca-signed-certificates-in-a-keystore.md:1]: Fixed with https://github.com/wso2/docs-is/pull/4839

**Pull request merged**